### ISSUE TITLE
Please fix `minimal.c` example.

### ISSUE THREAD

**Issue by define-private-public** [mypaint/libmypaint#113]
Hi.  I'm interested in writing some C# bindings for this.  I was doing a test port of the `minimal.c` example and I ran into two problems:

1. The calls to `stroke_to` are missing their `viewzoom` and `viewrotation` arguments.  What would be correct ones? for now I'm passing in `1.0` and `0.0` respectively
2. Under the hood, it looks like I was able stroke some lines, but I can't seem to save the output with the `mypaint_surface_save_png()` function.  Can you fixup the PPM saving function instead?

**Comment by briend** [comment]
I added the missing arguments, 1.0 and 0.0 are good values representing 100% zoom and no canvas rotation.  I don't know about the write_ppm/png issue though

**Comment by jplloyd** [comment]
Minimal example is fixed as of this commit: f0f2c6f998d771789b0b65bed128e6e01d9b1314
From the `examples` folder, compile it with: 
```
gcc minimal.c -I.. -I/usr/include/glib-2.0 -I/usr/include/json-c \
-lglib-2.0 -lgobject-2.0 -lm -ljson
```
_(adjust include paths based on your system)_
